### PR TITLE
[Spaces] Doc about setting custom sleep time 😴

### PR DESCRIPTION
**Pull request by julien-c**
Sorry about the few unrelated changes, hopefully it's ok

### CODE REVIEW

File: docs/hub/spaces-gpus.md [hunk 1]
```diff
@@ -91,4 +91,26 @@ Spaces running on free hardware are suspended automatically if they are not used
...
\ No newline at end of file
+Additional information about billing can be found in the [dedicated Hub-wide section](./billing).
+
+## Expose a custom sleep time[[sleep-time]]
+
+If your Space runs on the default `cpu-basic` hardware, it will go to sleep if inactive for more than a set time (currently, 72 hours). Anyone visiting your Space will restart it automatically.
+
+If you want your Space to never deactivate, or if you want to set a custom sleep time, you need to upgrade to a paid Hardware.
+
+By default, an upgraded Space will never go to sleep. However, you can use this setting for your upgraded Space to "turn off" (go back to cpu-basic) when it's unused. 😴
```
Comment (apolinario) [inline docs/hub/spaces-gpus.md:102]: Is it what is happening? Going back to `cpu-basic` when inactive? I thought it was in that inactive state that gets rebuilt once a visitor comes just like the 72h

File: docs/hub/spaces-changelog.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Spaces Changelog
 
+
+## [2022-12-07] - Add support for Streamlit 1.15
```
Comment (osanseviero) [inline docs/hub/spaces-changelog.md:4]: Also add another for the custom sleep time

File: docs/hub/spaces-config-reference.md [hunk 3]
```diff
@@ -32,6 +32,10 @@ Streamlit versions are supported from `0.79.0` to `1.10.0`.
 Path to your main application file (which contains either `gradio` or `streamlit` Python code, or `static` html code).  
 Path is relative to the root of the repository.  
 
+**`fullWidth`**: _boolean_  
+Whether your Space is rendered inside a full-width (when `true`) or fixed-width column (ie. "container" CSS) inside the iframe.
+Defaults to false in gradio and streamlit, and to true for other sdks.
```
Comment (osanseviero) [inline docs/hub/spaces-config-reference.md:37]:
```suggestion
Defaults to false in `gradio` and `streamlit`, and to true for other sdks.
```

File: docs/hub/spaces-gpus.md (hunk 1 above)
Comment (apolinario) [inline docs/hub/spaces-gpus.md:102]:
```suggestion
By default, an upgraded Space will never go to sleep. However, you can use this setting for your upgraded Space to become idle when it's unused 😴. The Space will get restarted once it receives a new visitor.
```

**Review by apolinario — APPROVED** [review]

File: docs/hub/spaces-gpus.md [hunk 4]
```diff
@@ -91,4 +91,26 @@ Spaces running on free hardware are suspended automatically if they are not used
...
 
-Additional information about billing can be found in the [dedicated section](./billing).
\ No newline at end of file
+Additional information about billing can be found in the [dedicated Hub-wide section](./billing).
+
+## Expose a custom sleep time[[sleep-time]]
+
+If your Space runs on the default `cpu-basic` hardware, it will go to sleep if inactive for more than a set time (currently, 72 hours). Anyone visiting your Space will restart it automatically.
+
+If you want your Space to never deactivate, or if you want to set a custom sleep time, you need to upgrade to a paid Hardware.
```
Comment (osanseviero) [inline docs/hub/spaces-gpus.md:100]:
```suggestion
If you want your Space never to deactivate or if you want to set a custom sleep time, you need to upgrade to a paid Hardware.
```

**Review by osanseviero — APPROVED** [review]
Docs look good but I have same question as Poli

File: docs/hub/spaces-gpus.md (hunk 1 above)
Comment (apolinario) [reply docs/hub/spaces-gpus.md:102]: IMO it is important to make clear that people are not charged when the Space is sleeping

Comment (julien-c) [reply docs/hub/spaces-gpus.md:102]: i'm actually not sure 🤷 

but yeah i think you're right, it transitions to "stopped" stage, i'll update

File: docs/hub/spaces-changelog.md (hunk 2 above)
Comment (julien-c) [reply docs/hub/spaces-changelog.md:8]: bc2a6f111d5f06a097f8a82cf3654cd8612974ea

**Comment by julien-c** [comment]
okok i'm about to merge this in 5-10 mins unless objection!

File: docs/hub/spaces-more-ways-to-create.md [hunk 5]
```diff
@@ -6,4 +6,8 @@ The [Repo Duplicator](https://huggingface.co/spaces/osanseviero/repo_duplicator)
 
 ## Creating a Space from a model
 
-The [Model Comparator Space Builder](https://huggingface.co/spaces/farukozderim/Model-Comparator-Space-Builder) can be used to create a Space directly from any model hosted on the Hub.
+New! You can now create a Gradio demo directly from most model pages, using the "Deploy -> Spaces" button.
+
+<video src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/model-page-deploy-to-spaces.mp4" controls autoplay muted loop />
```
Comment (julien-c) [inline docs/hub/spaces-more-ways-to-create.md:11]: arf, this doesn't seem to work: 
image

@mishig25 when you're back let me know if you think of a way to enable this

**Pull request merged**